src/app/vulnerability-db/[id]/page.tsx

```diff
@@ -0,0 +1,10 @@
+import ArticleDetail from '@/app/vulnerability-db/_components/ArticleDetail';
+
+export default function VulnerabilityDbDetailPage({ params }: { params: { id: string } }) {
```

Comment:
```suggestion
type VulnerabilityDbDetailPageProps = {
  params: {
    id: string;
  };
}

export default function VulnerabilityDbDetailPage({ params: { id } }: VulnerabilityDbDetailPageProps) {
```
P5: 이렇게 구조분해 할당 해줘도 괜찮을 것 같기도 한데.. prop 타입 이름이 넘 길어서 별로인 것 같기두 하고 ?! 그렇네용..

Reply:
문득보다가 DB B도 약자라서 대문자로 써야 할거 같슴당